internal/version/version.go

```diff
@@ -33,7 +37,8 @@ func GetVersion() error {
 
 type Info struct {
 	GitVersion   string
-	GitCommit    string
```

Comment:
This is technically a breaking change. I assume we are not relying on it somewhere else?

Reply:
It's only used to print details when starting up (and the version endpoint, of course)